tests/sanity/ignore-2.10.txt

```diff
@@ -1,28 +1 @@
-plugins/modules/mso_label.py validate-modules:doc-required-mismatch
-plugins/modules/mso_role.py validate-modules:doc-required-mismatch
-plugins/modules/mso_role.py validate-modules:parameter-list-no-elements
-plugins/modules/mso_schema.py validate-modules:doc-required-mismatch
-plugins/modules/mso_schema.py validate-modules:parameter-list-no-elements
-plugins/modules/mso_schema_site.py validate-modules:doc-required-mismatch
-plugins/modules/mso_schema_site_anp_epg.py validate-modules:doc-required-mismatch
-plugins/modules/mso_schema_site_anp_epg_domain.py pylint:ansible-bad-function
-plugins/modules/mso_schema_site_anp_epg_domain.py validate-modules:doc-required-mismatch
-plugins/modules/mso_schema_site_anp_epg_staticleaf.py validate-modules:doc-required-mismatch
-plugins/modules/mso_schema_site_anp_epg_staticport.py validate-modules:doc-required-mismatch
-plugins/modules/mso_schema_site_anp_epg_subnet.py validate-modules:doc-required-mismatch
-plugins/modules/mso_schema_site_bd_l3out.py validate-modules:doc-required-mismatch
-plugins/modules/mso_schema_site_vrf_region.py validate-modules:doc-required-mismatch
-plugins/modules/mso_schema_site_vrf_region_cidr.py validate-modules:doc-required-mismatch
-plugins/modules/mso_schema_site_vrf_region_cidr_subnet.py validate-modules:doc-required-mismatch
-plugins/modules/mso_schema_template_anp_epg.py validate-modules:parameter-list-no-elements
-plugins/modules/mso_schema_template_bd.py validate-modules:parameter-list-no-elements
-plugins/modules/mso_schema_template_contract_filter.py validate-modules:invalid-ansiblemodule-schema
-plugins/modules/mso_schema_template_contract_filter.py validate-modules:parameter-list-no-elements
-plugins/modules/mso_schema_template_deploy.py validate-modules:doc-required-mismatch
-plugins/modules/mso_schema_template_filter_entry.py validate-modules:parameter-list-no-elements
-plugins/modules/mso_site.py validate-modules:doc-required-mismatch
-plugins/modules/mso_site.py validate-modules:parameter-list-no-elements
-plugins/modules/mso_tenant.py validate-modules:doc-required-mismatch
-plugins/modules/mso_tenant.py validate-modules:parameter-list-no-elements
-plugins/modules/mso_user.py validate-modules:doc-required-mismatch
-plugins/modules/mso_user.py validate-modules:parameter-list-no-elements
\ No newline at end of file
+plugins/modules/mso_schema_template_bd.py validate-modules:invalid-ansiblemodule-schema
```

Comment:
Great to see these being fixed, thanks :)

Reply:
Any idea why I am having to still keep this invalid-ansiblemodule-schema because it seems to complain about a suboption inside a suboption item. Is that not supported?